pkg/cloud/services/cloudformation/bootstrap.go

```diff
@@ -296,6 +296,28 @@ func cloudProviderControlPlaneAwsPolicy() *iam.PolicyDocument {
 					"kms:DescribeKey",
 				},
 			},
+			{
+				Effect:   iam.EffectAllow,
+				Resource: iam.Resources{"*"},
+				Action: iam.Actions{
+					"ec2:CreateNetworkInterface",
+					"ec2:AttachNetworkInterface",
+					"ec2:DeleteNetworkInterface",
+					"ec2:DetachNetworkInterface",
+					"ec2:DescribeNetworkInterfaces",
+					"ec2:DescribeInstances",
+					"ec2:ModifyNetworkInterfaceAttribute",
+					"ec2:AssignPrivateIpAddresses",
+					"ec2:UnassignPrivateIpAddresses",
+				},
+			},
+			{
+				Effect:   iam.EffectAllow,
+				Resource: iam.Resources{"arn:aws:ec2:*:*:network-interface/*"},
+				Action: iam.Actions{
+					"ec2:CreateTags",
+				},
+			},
```

Comment:
Would it make sense to make optional?  Amazon best practices is to grant [Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).

The question is how to make this optional? Would this be best exposed in `clusterawsadm` as a flag?

Maybe `--cni amazon-vpc-cni`.  Not sure there are other CNI's that would require different policy.

or `--add-vpc-cni-policy`

Thoughts on if this should be optional and if so, how best to expose?

Reply:
someone wants to create mutil clusters with different network solution, such as calico and the amazon-vpc-cni, under one AWS account. 
that use case how to separate the privilege of account ?

Reply:
Make it accept multi-value with comma-separation?

Options:
- Calico
- AmazonVPC

Default: [Calico]

Reply:
agree with you, we can talk about this issues later.